### PR TITLE
Matomo modal update

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

**Comment by LianaHus** [comment]
I would not add the part explaining what is opt in. Everything else is ok

File: apps/remix-ide/src/app.js [hunk 1]
```diff
@@ -368,13 +368,13 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
   // Ask to opt in to Matomo for remix, remix-alpha and remix-beta
   if (window.location.hostname.includes('.ethereum.org') && !registry.get('config').api.exists('settings/matomo-analytics')) {
     modalDialog(
-      'Help us to improve our IDE',
+      'Help us to improve Remix IDE',
       yo`
       <div>
-        <p>Remix IDE uses <a href="https://matomo.org" target="_blank">Matomo</a>, an open source data analytics platform, to improve our website. Matomo on Remix is opt-in - meaning that we won't collect any information unless you agree.</p>
-        <p>We realize that our users have sensitive information in their code and that the privacy of our users must be protected.</p>
+        <p>An Opt-in version of <a href="https://matomo.org" target="_blank">Matomo</a>, an open source data analytics platform is being used to improve the Remix IDE website.</p>
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app.js:374]: I would choose `Remix IDE platform` instead of `Remix IDE website`

File: apps/remix-ide/src/app.js [hunk 2]
```diff
@@ -368,13 +368,13 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
...
+        <p>We realize that our users have sensitive information in their code and that their privacy - your privacy - must be protected.</p>
         <p>All data collected through Matomo is stored on our own server - no data is ever given to third parties.  Our analytics reports are public: <a href="https://matomo.ethereum.org/index.php?module=MultiSites&action=index&idSite=23&period=day&date=yesterday" target="_blank">take a look</a>.</p>
-        <p>We do not store any personally identifiable information (PII).</p>
+        <p>We do not collect nor store any personally identifiable information (PII).</p>
         <p>For more info see: <a href="https://medium.com/p/66ef69e14931/" target="_blank">Matomo Analyitcs on Remix iDE</a>.</p>
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app.js:378]: If possible, lets add a Remix IDE publication medium link here.

Comment (Aniket-Engg) [inline apps/remix-ide/src/app.js:378]: May be a comma after `For more info`

File: apps/remix-ide/src/app.js (hunk 1 above)
Comment (LianaHus) [reply apps/remix-ide/src/app.js:374]: Or just Remix IDE

File: apps/remix-ide/src/app.js (hunk 2 above)
Comment (ryestew) [reply apps/remix-ide/src/app.js:378]: Do you think we need the link to the medium article here on line 377? - it is on the next line.

**Review by LianaHus — APPROVED** [review]

**Pull request merged**